setup.py

```diff
@@ -5,74 +5,84 @@
 # This file is part of the jupyter-fs library, distributed under the terms of
 # the Apache License 2.0.  The full license can be found in the LICENSE file.
 #
-from setuptools import setup, find_packages
 from codecs import open
 from os import path
+from pathlib import Path
+from setuptools import setup, find_packages
+from subprocess import CalledProcessError
 
 from jupyter_packaging import (
-    create_cmdclass, install_npm, ensure_targets,
-    combine_commands, ensure_python, get_version
+    combine_commands, command_for_func, create_cmdclass, ensure_python,
+    ensure_targets, get_version, run
 )
 
-pjoin = path.join
-
 ensure_python(('2.7', '>=3.3'))
 
 name = 'jupyter-fs'
 here = path.abspath(path.dirname(__file__))
-version = get_version(pjoin(here, "jupyterfs", '_version.py'))
+version = get_version(path.join(here, "jupyterfs", '_version.py'))
 
 with open(path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
 requires = [
     'fs>=2.4.11',
-    'jupyterlab>=1.0.0',
+    'fs-s3fs>=1.1.1',
+    'jupyterlab>=2.0.0',
     'notebook>=5.7.0',
 ]
 
 dev_requires = requires + [
     'autopep8',
+    'boto3',
     'bump2version',
     'codecov',
     'flake8',
+    'fs-miniofs',
     'mock',
     'pylint',
     'pytest',
     'pytest-cov',
 ]
 
-data_spec = [
+data_files_spec = [
     # Lab extension installed by default:
     ('share/jupyter/lab/extensions',
      'lab-dist',
-     'multicontentsmanager-*.tgz'),
+     'jupyter-fs-*.tgz'),
     # Config to enable server extension by default:
     ('etc/jupyter',
      'jupyter-config',
      '**/*.json'),
 ]
 
 
-cmdclass = create_cmdclass('js', data_files_spec=data_spec)
-cmdclass['js'] = combine_commands(
-    install_npm(here, build_cmd='build:all'),
+def runPackLabextension():
+    if Path('package.json').is_file():
+        try:
+            run(['jlpm', 'build:all'])
+        except CalledProcessError:
+            pass
+
```

Comment:
`pack_labext` was unused

Reply:
erg. That should have been:

```
def runPackLabextension():
    if Path('package.json').is_file():
        try:
            run(['jlpm', 'build:all'])
        except CalledProcessError:
            pass
pack_labext = command_for_func(runPackLabextension)

cmdclass = create_cmdclass('pack_labext', data_files_spec=data_files_spec)
cmdclass['pack_labext'] = combine_commands(
    pack_labext,
    ensure_targets([
        path.join(here, 'lib', 'index.js'),
        path.join(here, 'style', 'index.css')
    ]),
)
```

I'm fine with leaving this as a moot point